### ISSUE TITLE
Accessibility - Individual Country Screen - Talkback Reading Entire Country Details When Hidden Off Screen

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#34]
TalkBack continued to read the rest to the detail text on the page even though it was not visible.

**Comment by aquilent-dgemoets** [comment]
This is the same behavior as the iOS version, and appears to be desired behavior.

**Comment by jfc3-dol** [comment]
This is an open issue with the iOS version of the application as far as I can remember. The application should only read the last visible text and then read the 'More' link. It's helpful for screen reader users that it keeps reading, but my guess it's not the desired experience.

**Comment by jfc3-dol** [comment]
Not sure why TalkBack like VoiceOver in iOS reads the text hidden off screen.

**Comment by jfc3-dol** [comment]
My guess is the content for both iOS and Android are visually off-screen instead of using **display: none** as a way to hide the content. Keeping issue open since it still is a problem.

**Comment by jfc3-dol** [comment]
Even though the majority of the content is not displayed the screen reader user has to listen to all of it because it's only visually off-screen and not readable by the screen reader.

Again this might be an Android issue that can't be resolved and left open to check with each new version of the application.

**Comment by NJICF** [comment]
Made "More" text not readable.

**Comment by jfc3-dol** [comment]
Okay, will have to test with TalkBack to see how it is read along with how it works for a visual user.

My guess is the "More" link can be activated by tapping on the link to show the rest of the hidden text.

**Comment by jfc3-dol** [comment]
Not necessarily the best way to have that work but it's nice not having TalkBack read the whole long part of the text and then a "More" link, and when tapped does not show/read more text because it's already been read.

Closing issue.
